xDrip/Scenes/Settings/SettingsChartRanges/SettingsChartRangesPresenter.swift

```diff
@@ -19,10 +19,195 @@ protocol SettingsChartRangesPresentationLogic {
 final class SettingsChartRangesPresenter: SettingsChartRangesPresentationLogic {
     weak var viewController: SettingsChartRangesDisplayLogic?
     
+    private lazy var settings: Settings = {
```

Comment:
Presenter shouldn't access models directly, they should be passed in `Response` by interactor.